nym-data-observatory/build.rs

```diff
@@ -0,0 +1,36 @@
+// Copyright 2025 - Nym Technologies SA <contact@nymtech.net>
+// SPDX-License-Identifier: Apache-2.0
+use glob::glob;
+use std::env;
+use std::path::Path;
+
+fn main() {
+    // copy common manifest files from "../common/nyxd-scraper-psql/sql_migrations/* migrations"
+    println!("Copying common migrations...");
```

Comment:
I don't remember from the top of my head, but are we allowed to have debug `println!` inside `build.rs`? I think the compiler is actually using the STDOUT, like with this: `println!("cargo:rustc-env=DATABASE_URL={database_url}");`

Reply:
it works